cli/src/commands/diffedit.rs

```diff
@@ -59,6 +59,9 @@ pub(crate) struct DiffeditArgs {
     /// Specify diff editor to be used
     #[arg(long, value_name = "NAME")]
     tool: Option<String>,
+    /// Do not modify the content of the revision specified with `-r` or `--to`.
```

Comment:
Nit: No trailing period in doc strings.

Reply:
Thanks. I've also cleaned up the rest of the help strings for `diffedit` in #4497.